pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -452,11 +452,12 @@ func (c *Controller) addNodeRoute(nodeName string, node *v1.Node) error {
 	}
 	peerGatewayIP := ip.NextIP(peerPodCIDRAddr)
 
+	ipsecTunOFPort := int32(0)
```

Comment:
you use an `int32` here but it seems that the rest of the code uses `uint32` pretty consistently (at least the OF client code does)?

Reply:
Hmm never mind, I see that the interface config uses an `int32`... so I guess we are not consistent.

Reply:
Yes. I think we should use int32, but I prefer not to change all ofclient code.